src/transformStateWithClones.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- *
- * @return {Object[]}
- */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const current = { ...state };
+  const resultingArray = [];
+  let newr;
```

Comment:
I don't understand this name

Reply:
Yep, it was awful decision to give that name to this, I will change it